src/openrct2-ui/windows/Land.cpp

```diff
@@ -424,7 +426,7 @@ namespace OpenRCT2::Ui::Windows
             {
                 gInputDragLast.y += tile_height;
 
-                SelectionRaiseLand(GAME_COMMAND_FLAG_APPLY);
+                SelectionRaiseLand(true);
```

Comment:
I can see why this was simplified, but I think the named flag carries more intuitive semantics than just an unnamed bool, tbh. Same below.

Reply:
It was the only flag that was ever passed through here. I can make a local enum class, though.